tests/test_api_equivalence.py

```diff
@@ -186,3 +186,46 @@ def test_api_equivalence_continued_simulation():
 
     v2 = jnp.concatenate([v21, v22[:, 1:]], axis=1)
     assert np.max(np.abs(v1 - v2)) < 1e-8
+
+
+def test_api_equivalence_network_matches_cell():
```

Comment:
isnt this redundant or does it matter how the jax.sparse deals with synapses?

Reply:
It's just to make very sure :)